cmd/apiserver-boot/boot/init_repo/repo.go

```diff
@@ -249,17 +249,37 @@ package apis
 `
 
 var workspaceTemplate = `
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "io_k8s_repo_infra",
+    sha256 = "5ee2a8e306af0aaf2844b5e2c79b5f3f53fc9ce3532233f0615b8d0265902b2a",
+    strip_prefix = "repo-infra-0.0.1-alpha.1",
+    urls = [
+        "https://github.com/kubernetes/repo-infra/archive/v0.0.1-alpha.1.tar.gz",
+    ],
+)
+
+# built-in versions of package rules will no longer be addressed
+# see https://docs.bazel.build/versions/1.1.0/be/pkg.html#deprecated
 http_archive(
-    name = "io_bazel_rules_go",
-    url = "https://github.com/bazelbuild/rules_go/releases/download/0.6.0/rules_go-0.6.0.tar.gz",
-    sha256 = "ba6feabc94a5d205013e70792accb6cce989169476668fbaf98ea9b342e13b59",
+    name = "rules_pkg",
+    urls = ["https://github.com/bazelbuild/rules_pkg/releases/download/0.2.4/rules_pkg-0.2.4.tar.gz"],
+    sha256 = "4ba8f4ab0ff85f2484287ab06c0d871dcb31cc54d439457d28fd4ae14b18450a",
 )
```

Comment:
I mean you should remove the 5 lines about the `http_archive` of `rules_pkg`.

Reply:
thx for clarification